### PR TITLE
Added more support for spanish-based tests

### PR DESCRIPTION
**Pull request by downvec**
Ported functionality from @amunoz/legibilidad in reference to #109

### CODE REVIEW

**Comment by alxwrd** [comment]
@downvec thanks for this. I've just pushed a commit to remove an f string where there shouldn't be one so the build passes.

I'll give the a proper review later.

File: textstat/textstat.py [hunk 1]
```diff
@@ -521,7 +521,79 @@ def reading_time(self, text, ms_per_char=14.69):
...
+        Fernandez Huerta readability score
+        https://legible.es/blog/lecturabilidad-fernandez-huerta/
+        '''
+        sentence_lenth = self.avg_sentence_length(text)
+        syllables_per_word = self.avg_syllables_per_word(text)
+
+        f_huerta = (
+            206.85 - float(60 * syllables_per_word) -
+            float(1.02 * sentence_lenth))
+        return legacy_round(f_huerta, 1)
```
Comment (alxwrd) [inline textstat/textstat.py:537]:
```suggestion
        sentence_length = self.avg_sentence_length(text)
        syllables_per_word = self.avg_syllables_per_word(text)

        f_huerta = (
            206.85 - float(60 * syllables_per_word) -
            float(1.02 * sentence_length))
        return legacy_round(f_huerta, 1)
```

**Comment by downvec** [comment]
- [x]  The functions purpose and interpretation has been added to README.md
- [x] Each function has been tested

**Comment by alxwrd** [comment]
See [0.7.0](https://github.com/shivam5992/textstat/releases/tag/0.7.0)

**Pull request merged**